maintenance_equipment_status/views/maintenance_equipment_views.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- equipment : views -->
+    <record id="hr_equipment_view_form" model="ir.ui.view">
+        <field name="name">equipment.form</field>
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_form"/>
+        <field name="arch" type="xml">
+            <sheet position="before">
+                <header>
```

Comment:
@etobella You mean this header?

Reply:
Yes

Reply:
Ok, I agree.

@NuriaMartinXifre can you migrate base_maintenance and include the empty header for this view in that module? then, we'll need to include it here as a dependency and extend the view accordingly.